tutorials/animation/creating_movies.rst

```diff
@@ -296,6 +297,22 @@ detail and reduce light leaking:
             get_viewport().world_3d.environment.sdfgi_min_cell_size *= 0.25
             get_viewport().world_3d.environment.sdfgi_cascades = 8
 
+ .. code-tab:: csharp
+
+    public partial class MyNode3D : Node3D
```

Comment:
Since this looks like the complete script file, we should include the `using` statements.

```suggestion
    using Godot;

    public partial class MyNode3D : Node3D
```